nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -57,6 +57,64 @@
       and <option>services.xserver.desktopManager.xfce4-14</option> simultaneously or to downgrade from Xfce 4.14 after upgrading.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The GNOME 3 desktop manager module sports an interface to enable/disable core services, applications, and optional GNOME packages
+      like games.
+      <itemizedlist>
+      <para>This can be achieved with the following options which the desktop manager default enables</para>
+      <listitem><para><link linkend="opt-services.gnome3.core-os-services.enable"><literal>services.gnome3.core-os-services.enable</literal></link></para></listitem>
```

Comment:
Cannot the `link` content be filled in automatically?

Reply:
Doesn't appear so.

Reply:
Does not `<xref linkend="opt-services.gnome3.core-os-services.enable"/>`

Reply:
https://github.com/NixOS/nixpkgs/pull/68196